assets/pages/oneoff-contributions/helpers/ajax.js

```diff
@@ -66,13 +67,13 @@ function requestData(
   const { user } = getState().page;
 
   if (user.fullName !== null && user.fullName !== undefined &&
-    user.email !== null && user.email !== undefined) {
+    user.email !== null && user.email !== undefined && token.paymentMethod === 'Stripe') {
```

Comment:
Unrelated, but when reading this file is noticed `user.fullName` isn't used – that's why it is missing in the new version.